### PR TITLE
AWS BnR Tech Note for Secret starting with `-`

### PR DESCRIPTION
**Pull request by prat98**

### CODE REVIEW

File: stories/topics/tech-note-aws-backup-restore.adoc [hunk 1]
```diff
@@ -22,6 +22,16 @@
 * `<stack_name>-pulp_cert`
 * `<stack_name>-pulp_key`
 
+. The Ansible on Clouds Ops Container restore playbook fails when attempting to restore a secret starting with `-`. If the restore operation is failing, on your deployment please ensure the secret `<stack_name>-database_fields_symmetric_key` does not start with a `-`. Follow the steps below to check for this case and update the secret appropriately.
```
Comment (ianf77) [inline stories/topics/tech-note-aws-backup-restore.adoc:25]: If this is a known issue, it ought also to be in the Release Notes under 'Known issues'

File: stories/topics/tech-note-aws-backup-restore.adoc [hunk 2]
```diff
@@ -22,6 +22,16 @@
...
+.. Find the backup you are passing to the restore playbook on S3.
+... The `aws_backup_name` parameter you passed to the restore playbook is the backup we are trying to locate and it should be a folder inside of an S3 bucket
```
Comment (ianf77) [inline stories/topics/tech-note-aws-backup-restore.adoc:27]: . The aws-backup name ...

File: stories/topics/tech-note-aws-backup-restore.adoc [hunk 3]
```diff
@@ -22,6 +22,16 @@
...
+... The `aws_s3_bucket` parameter you passed to the restore playbook should be the bucket which contains the backup we are trying to locate
```
Comment (ianf77) [inline stories/topics/tech-note-aws-backup-restore.adoc:28]: . The aws_s3_bucket ...

File: stories/topics/tech-note-aws-backup-restore.adoc [hunk 4]
```diff
@@ -22,6 +22,16 @@
...
+.. After locating the backup, navigate inside of the backup folder and download the `secrets.json` file.
```
Comment (ianf77) [inline stories/topics/tech-note-aws-backup-restore.adoc:29]: One period for each of these.

**Review by ianf77 — COMMENTED** [review]
A few  comments.

File: stories/topics/tech-note-aws-backup-restore.adoc (hunk 4 above)
Comment (prat98) [reply stories/topics/tech-note-aws-backup-restore.adoc:29]: Screenshot 2023-07-27 at 11 59 20 AM

**Pull request merged**